ibis/expr/types/pretty.py

```diff
@@ -139,12 +140,11 @@ def _(dtype, values):
                 v = "".join(
                     f"[dim]{repr(c)[1:-1]}[/]" if not c.isprintable() else c for c in v
                 )
-            elif len(v) <= max_string:
-                url = urlparse(v)
-                # check both scheme and netloc to avoid rendering e.g.,
-                # `https://` as link
-                if url.scheme and url.netloc:
-                    v = f"[link]{v}[/link]"
+            url = urlparse(raw_v)
```

Comment:
Maybe I'm misunderstanding, but I didn't see the need for the previous `elif` check, so I removed it. please correct me if I missed something here.